SparkPost/2web2ui
# Sa 1896-2

### What Changed

- Added error state handling

### How To Test

- Go to /signals/schedule/foo and /signals/schedule/foo/bar. Verify that you are redirected back to report builder.
